distribution/docs/src/main/resources/content/_architectures/_securityFramework/security-framework-intro.adoc

```diff
@@ -1,10 +1,11 @@
 :title: Security Framework
 :type: securityFrameworkIntro
 :status: published
+:children: Subject, Security Core, Security Encryption, Security LDAP, Security PDP, Web Service Security Architecture, Security PEP, Filtering, Expansion Service, Federated Identity, Realms
 :order: 00
 :summary: Introduction to Security Framework.
 
 == {title}
 
-The ${branding} Security Framework utilizes http://shiro.apache.org/[Apache Shiro] as the underlying security framework.
+The ${branding} Security Framework builds off of the interfaces provided by http://shiro.apache.org/[Apache Shiro] to create an underlying security framework that developers might be familiar with.
```

Comment:
❓ Is there anyway to make it clearer that we only use a small part of Shiro to keep people from going to their documentation and getting confused?